### ISSUE TITLE
Create machine learning script

### ISSUE THREAD

**Issue by jmcollie** [jmcollie/disaster_response_pipeline#2]
A python script is needed to take the database file path and model file path, create and train a classifier, and store the classifier into a pickle file to the specified model file path.

- The script should use a custom tokenize function using nltk to case normalize, lemmatize, and tokenize text. This function is used in the machine learning pipeline to vectorize and then apply TF-IDF to the text.
- The script builds a pipeline that processes text and then performs multi-output classification on the 36 categories in the dataset. GridSearchCV is used to find the best parameters for the model.
- The TF-IDF pipeline should only be trained with the training data. The f1 score, precision and recall for the test set is outputted for each category.

**Comment by UzairURK** [comment]
Hi,
@jmcollie,
Hope you are fine. 
I want to work on this issue, can you assign this to me, So I can start working on it right away!
